sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -556,5 +556,201 @@ public static RecognizeLinkedEntitiesResultCollection RecognizeLinkedEntitiesRes
+        /// </summary>
+        /// <param name="result">Sets the <see cref="TextAnalytics.RecognizeLinkedEntitiesResultCollection"/> property.</param>
+        /// <param name="completedOn">Sets the <see cref="DateTimeOffset"/> property.</param>
+        /// <param name="code">Sets the code property.</param>
+        /// <param name="message">Sets the message property.</param>
+        /// <returns>A new instance of <see cref="TextAnalytics.RecognizeLinkedEntitiesActionResult"/> for mocking purposes.</returns>
+        public static RecognizeLinkedEntitiesActionResult RecognizeLinkedEntitiesActionResult(
+            RecognizeLinkedEntitiesResultCollection result,
+            DateTimeOffset completedOn,
+            string code,
+            string message)
+        {
+            return new RecognizeLinkedEntitiesActionResult(result, completedOn, new TextAnalyticsErrorInternal(code, message));
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="TextAnalytics.RecognizeEntitiesActionResult"/> for mocking purposes.
+        /// </summary>
+        /// <param name="result">Sets the <see cref="TextAnalytics.RecognizeEntitiesResultCollection"/> property.</param>
+        /// <param name="completedOn">Sets the <see cref="DateTimeOffset"/> property.</param>
+        /// <param name="code">Sets the code property.</param>
+        /// <param name="message">Sets the message property.</param>
+        /// <returns>A new instance of <see cref="TextAnalytics.RecognizeEntitiesActionResult"/> for mocking purposes.</returns>
+        public static RecognizeEntitiesActionResult RecognizeEntitiesActionResult(
+            RecognizeEntitiesResultCollection result,
+            DateTimeOffset completedOn,
+            string code,
+            string message)
+        {
+            return new RecognizeEntitiesActionResult(result, completedOn, new TextAnalyticsErrorInternal(code, message));
+        }
+
+        #endregion Action Result Models
+
+        #region Healthcare
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="TextAnalytics.AnalyzeHealthcareEntitiesResult"/> for mocking purposes.
+        /// </summary>
+        /// <param name="id">Sets the id property.</param>
+        /// <param name="statistics">Sets the <see cref="TextAnalytics.TextDocumentStatistics"/> property.</param>
+        /// <param name="healthcareEntities">Sets the collection of <see cref="TextAnalytics.HealthcareEntity"/> property.</param>
+        /// <param name="entityRelations">Sets the collection of <see cref="TextAnalytics.HealthcareEntityRelation"/> property.</param>
+        /// <param name="warnings">Sets the collection of <see cref="TextAnalytics.TextAnalyticsWarning"/> property.</param>
+        /// <returns>A new instance of <see cref="TextAnalytics.AnalyzeHealthcareEntitiesResult"/> for mocking purposes.</returns>
+        public static AnalyzeHealthcareEntitiesResult AnalyzeHealthcareEntitiesResult(
```

Comment:
same comment here about two methods because this one could have error information